frontend/src/pages/NewPlacePage.vue

```diff
@@ -377,14 +382,49 @@ export default {
             ]
         };
     },
+
+    created() {
+        this.$store.dispatch('placeCategory/getAllCategories')
+            .then((result) => {
+                this.categories = result;
+            });
+    },
+    
+    watch: {
+        'newPlace.category': function (categoryObject) {
+            if (categoryObject) {
+                this.newPlace.category_tags = [];
+                // this.$store.dispatch('placeCategoryTags/getTagsByCategoryId', categoryObject.id)
+                //     .then((result) => {
+                //         this.category_tags = result;
+                //     });
+                this.selectedTag = 'v';
+            }
+        },
+
+        'selectedTag': function (tagObject) {
+            if (tagObject !== 'v' && !this.newPlace.category_tags.some((tag) => tag.name === tagObject.name)) {
```

Comment:
another method `this.newPlace.category_tags.some((tag) => tag.name === tagObject.name)`